routes/api.php

```diff
@@ -27,12 +27,10 @@
     echo 'OpenTransportManager API';
 });
 
-Route::fallback(function (): RedirectResponse {
+Route::get('/docs', function (): RedirectResponse {
```

Comment:
Maybe it would be a good idea to not litter your routing and move this redirect into `DocsControllers` or something?